css-isolation/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "css-isolation",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "./app1",
+      "./app2"
+    ]
+  },
   "scripts": {
     "start": "lerna run --scope=css-isolation_app* --parallel start",
     "build": "lerna run --scope=css-isolation_app* --parallel build",
```

Comment:
And As I think we don't need lock file here. We should install all deps from the root of the repo and have only root .lock file